Sources/ArcGISToolkit/Components/Augmented Reality/FlyoverSceneView.swift

```diff
@@ -19,9 +19,12 @@ import ArcGIS
 /// A scene view that provides an augmented reality fly over experience.
 @MainActor
 @preconcurrency
+@available(visionOS, unavailable)
 public struct FlyoverSceneView: View {
+#if !os(visionOS)
```

Comment:
Why are these compiler directives (here and elsewhere) needed when the type itself is unavailable on visionOS?

Reply:
I wondered that myself when I started this. Even if the type is marked as unavailable and will never be used on visionOS, the compiler will still gives errors on the areas that are unavailable or not visible on visionOS.

Reply:
I see. Following your instructions, I got a build with visionOS support and I understand now what the problem is. Since the relevant types are marked as available on iOS rather than unavailable on visionOS:

![image](https://github.com/user-attachments/assets/f6655bb3-d719-4eaa-898d-06fb1db341cf)

I suggest changing the compiler directive to `os(iOS)` instead of `!os(visionOS)`.